src/server/service/kv.rs

```diff
@@ -247,6 +248,12 @@ macro_rules! set_total_time {
 impl<E: Engine, L: LockManager, F: KvFormat> Tikv for Service<E, L, F> {
     handle_request!(kv_get, future_get, GetRequest, GetResponse, has_time_detail);
     handle_request!(kv_scan, future_scan, ScanRequest, ScanResponse);
+    handle_request!(
```

Comment:
`tikv` service should focus on high level txn kv API. `kv_scan_txn_entry` exposes low level implementation details (txn data structures), how about moving it to `cdc` service?